README.md

```diff
@@ -31,6 +31,61 @@ SolidusImporter::ProcessImport.new(import).process
 
 To process in background: `SolidusImporter::ImportJob.perform_later(import.id)`
 
+### Accepted Format
+
+The accepted format is the [Shopify CSV](https://help.shopify.com/en/manual/products/import-export/using-csv) for which is also relatively easy to find exporters for every major platform (e.g. [shopify_transporter](https://github.com/Shopify/shopify_transporter)).
+
+There are three supported CSV types:
+
+1. [Product](https://help.shopify.com/en/manual/migrating-to-shopify/transporter-app/csv-products)
+2. [Order](https://help.shopify.com/en/manual/migrating-to-shopify/transporter-app/csv-orders)
+3. [Customer](https://help.shopify.com/en/manual/migrating-to-shopify/transporter-app/csv-customers)
+
+### The Processors
+
+The importing is managed by a list of processors for each CSV type, the default processors are:
+
+```rb
+Spree.config do |config|
+  config.importer.product_processors = %w[
+    SolidusImporter::Processors::Product
+    SolidusImporter::Processors::Variant
+    SolidusImporter::Processors::ProductImage
+    SolidusImporter::Processors::VariantImage
+    SolidusImporter::Processors::Log
+  ]
+  config.importer.order_processors = %w[
+    SolidusImporter::Processors::Order
+    SolidusImporter::Processors::Log
+  ]
+  config.importer.customer_processors = %w[
+    SolidusImporter::Processors::Customer
+    SolidusImporter::Processors::Log
+  ]
```

Comment:
This configuration is not what we ended up with, we should either amend the current configuration class to separate the config for each processor and/or update this section of the readme.

Reply:
I've updated the README to fit the current implementation. The configurations improvements will come in a separate PR.